server/lib/txgh-server/transifex_request_auth.rb

```diff
@@ -1,22 +1,32 @@
+require 'json'
 require 'openssl'
 require 'base64'
 
 module TxghServer
   class TransifexRequestAuth
-    HMAC_DIGEST = OpenSSL::Digest.new('sha1')
-    RACK_HEADER = 'HTTP_X_TX_SIGNATURE'
-    TRANSIFEX_HEADER = 'X-TX-Signature'
+    HMAC_DIGEST = OpenSSL::Digest.new('sha256')
```

Comment:
Mentioned in person, TXGH upstream contains a either version of this class(i.e. it works for either version). Should fix tests but we won't be testing against production

Reply:
I don't think they use the old signature strategy anymore, so this should work in production too.